docs/set_up_admin_tails.rst

```diff
@@ -55,17 +55,14 @@ Start by running the following commands to download the git repository.
 Verify the Release Tag
 ~~~~~~~~~~~~~~~~~~~~~~
 
-First, download the *Freedom of the Press Foundation Master Signing Key*
-and verify the fingerprint.
+First, download and verify the *Freedom of the Press Foundation Master
+Signing Key*. When passing the full public key fingerprint to the
+:code:`--recv-key` command, GPG will implicitly verify that the
```

Comment:
Fixed in https://github.com/fowlslegs/securedrop/commit/caca462e2b3fb764587d7d42c3e6d0971fb80981